MMCE_PHASE_A_PARITY.md

```
@@ -0,0 +1,163 @@
+  src/mmcesupport.c \
+  include/mmcesupport.h \
+  modules/iopcore/cdvdman/device-mmce.c \
+  modules/mcemu/device-mmce.c \
+  modules/iopcore/common/mmcedrv_config.h \
+  .github/patches/mmceman-fs-close-fd-leak.patch \
+  .github/patches/mmceman-fs-open-enoent.patch \
+  .github/patches/mmceman-fs-dopen-enoent.patch \
+  .github/scripts/install_coherent_mmce.sh
+```
+
+### A. Zero Diff (100% Bit-for-Bit Identical to `master`)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)